common-test/src/main/scala/org/genivi/sota/data/PackageIdGenerators.scala

```diff
@@ -22,6 +22,17 @@ trait PackageIdGenerators {
   val genPackageIdVersion: Gen[PackageId.Version] =
     Gen.listOfN(3, Gen.choose(0, 999)).map(_.mkString(".")).map(Refined.unsafeApply) // scalastyle:ignore magic.number
 
+  def genConflictFreePackageIdVersion(n: Int): Seq[PackageId.Version] = {
```

Comment:
Yes I thought of this. Something like the following might be useful: https://gist.github.com/etorreborre/d0616e704ed85d7276eb12b025df8ab0

But since in this instance I just want to test 2 versions, I thought I'd leave generalising it for a later date, when we actually have more need of it.